Classes/Controller/ToolboxController.php

```diff
@@ -130,6 +139,30 @@ private function renderToolByName(string $tool): void
         $this->view->assign($tool, true);
     }
 
+    /**
+     * Get the URL of the model.
+     *
+     * Gets the URL of the model by parameter or from the configured file group of the document.
+     *
+     * @access private
+     *
+     * @return string
+     */
+    private function getModelUrl(): string
+    {
+        $modelUrl = '';
+        if (!empty($this->requestData['model'])) {
+            $modelUrl = $this->requestData['model'];
+        } elseif (!($this->isDocMissingOrEmpty() || empty($this->settings['fileGrpsModelDownload']))) {
```

Comment:
Access to properties from `ext_conf_template.txt`:
Example: `GeneralUtility::trimExplode(',', $this->extConf['files']['fileGrpThumbs']);`

Reply:
Thx for this! I adjusted my implementation accordingly, removed some misleading comments, and eliminated `fileGrpsImageDownload` from `tx_dlf_pdfdownloadtool`